packages/components/src/components/primitives/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC } from 'react';
+import {
+  Accordion as ChakraAccordion,
+  AccordionItem,
+  AccordionButton as ChakraAccordionButton,
+  AccordionPanel,
+  AccordionIcon,
+} from '@chakra-ui/react';
+
+import {
+  AccordionProps,
+  AccordionButtonProps,
+  AccordionStaticMembers,
+} from './types';
+
+export const Accordion: FC<AccordionProps> & AccordionStaticMembers = (
+  props
+) => {
+  return <ChakraAccordion _expanded={{ bg: '#]]]' }} {...props} />;
```

Comment:
Is this `_expanded={{ bg: '#]]]' }}` needed?

Reply:
removed!